c10/test/util/intrusive_ptr_test.cpp

```diff
@@ -218,8 +218,6 @@ TEST(
     givenInvalidPtr_whenMoveAssigning_thenNewInstanceIsValid) {
   intrusive_ptr<SomeClass> obj1 = make_intrusive<SomeClass>();
   intrusive_ptr<SomeClass> obj2;
-  // NOLINTNEXTLINE(clang-analyzer-deadcode.DeadStores)
```

Comment:
I'm not sure the changes here are ok. The whole point of these tests is to observe the side effect of this code I think.

cc @ezyang

Reply:
I don't think `get()` has side effects? It just returns the pointer.

Reply:
But the `.weak` below do right?

Reply:
Oh you're right. Thanks, let me add those and the locks back in.

Reply:
Thanks, back in.